extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/GrpcServerRecorder.java

```diff
@@ -432,4 +464,17 @@ public void handle(AsyncResult<Void> ar) {
             });
         }
     }
+
+    private class DevModeWrapper {
+        private final ClassLoader classLoader;
+
+        public DevModeWrapper(ClassLoader contextClassLoader) {
+            classLoader = contextClassLoader;
+        }
+
+        public void run(Runnable command) {
+            Thread.currentThread().setContextClassLoader(classLoader);
```

Comment:
Shouldn't we reset the original TCCL after the command is run?

Reply:
in practice, the original one is the one before dev mode reload. But in theory, it makes sense, I'll do it